frontend/docs/intro.md

```diff
@@ -32,37 +32,169 @@ The Final Space API is maintained as an open source project on [GitHub](https://
+    "path": "/",
+    "fullUrl": "https://finalspaceapi.com/api/v0/"
+  },
+  {
+    "type": "GET",
+    "name": "All Characters",
+    "path": "/character",
+    "queryParams": [
+      {
+        "optional": true,
+        "name": "Sort By",
+        "values": [
+          "asc",
+          "desc"
+        ]
+      }
+    ],
+    "fullUrl": "https://finalspaceapi.com/api/v0/character"
+  },
+  {
+    "type": "GET",
+    "name": "Single Character",
+    "path": "/character/<id>",
+    "fullUrl": "https://finalspaceapi.com/api/v0/character/<id>"
+  },
+  {
+    "type": "GET",
+    "name": "All Episodes",
+    "path": "/episode",
+    "queryParams": [
+      {
+        "optional": true,
+        "name": "Sort By",
+        "values": [
+          "asc",
+          "desc"
+        ]
+      }
+    ],
+    "fullUrl": "https://finalspaceapi.com/api/v0/episode"
+  },
+  {
+    "type": "GET",
+    "name": "Single Episode",
+    "path": "/episode/<id>",
+    "fullUrl": "https://finalspaceapi.com/api/v0/episode/<id>"
+  }
+]
 ```
 
 
 :::info Endpoints
 
 
-There are currently two available resources:
+There are currently four available resources:
 
 - [Character](/docs/character): used to get all the characters.
   
+```
+GET https://finalspaceapi.com/api/v0/character
+```
+*Sample Request*
+
 ```
 https://finalspaceapi.com/api/v0/character
 ```
 
+*Sample Response*
+```json
+[{"id":1,"name":"Gary Goodspeed","status":"Alive","species":"Human","gender":"Male","hair":"Blonde",
+"alias":["The Gary(by Lord Commander)","Thunder Bandit(code name)","Star Dragon(new code name)",
+"Primate(By Clarence)"],"origin":"Earth","abilities":["Piloting","Marksmanship"],
+"img_url":"https://finalspaceapi.com/api/character/avatar/gary_goodspeed.jpg"},
+{"id":2,"name":"Mooncake","status":"Alive","species":"Mooncake's Species",
+"gender":"Male","hair":"None","alias":["Specimen E - 351","Little Buddy","Planet Destroyer","Dude cake","Little Guy"],
+"origin":"Created when John Goodspeed used an Anti-matter bomb to close a breach to Final Space.",
+"abilities":["Hovering","Firing Laser Beams","Planet Destruction"],
+"img_url":"https://finalspaceapi.com/api/character/avatar/mooncake.jpg"}]
+```
+
+
+- [Character with Character id](/docs/character): used to get a specific character details.
+  
+```
+GET https://finalspaceapi.com/api/v0/character/<character_id>
+```
+*Sample Request*
+
+```
+https://finalspaceapi.com/api/v0/character/2
+```
+
+*Sample Response*
+```json
+{"id":2,"name":"Mooncake","status":"Alive","species":"Mooncake's Species","gender":"Male","hair":"None",
+"alias":["Specimen E - 351","Little Buddy","Planet Destroyer","Dude cake","Little Guy"],
+"origin":"Created when John Goodspeed used an Anti-matter bomb to close a breach to Final Space.",
+"abilities":["Hovering","Firing Laser Beams","Planet Destruction"],
+"img_url":"https://finalspaceapi.com/api/character/avatar/mooncake.jpg"}
+```
+
+
 - [Episode](/docs/episode): used to get all the episodes.
```

Comment:
Only one example is enough for the intro.

Separate documentation for the endpoints cover this
So you can remove this.

Thanks